samples/README.md

```diff
@@ -268,6 +268,12 @@ Demonstrate how to do conditional rendering, dynamically discarding rendering co
 **Extension**: [```VK_EXT_vertex_input_dynamic_state```](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VK_EXT_vertex_input_dynamic_state.html) <br/>
 Demonstrate how to use vertex input bindings and attribute descriptions dynamically, which can reduce the number of pipeline objects that are need to be created.
 
+
+### [Extended dynamic state 2](./extensions/extended_dynamic_state2)
+**Extension**: [```VK_EXT_extended_dynamic_state2```](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VK_EXT_extended_dynamic_state2.html)
+<br/>
+Demonstrate how to use depth bias, primitive restart, rasterizer discard and patch control points dynamically, which can reduce the number of pipeline objects that are need to be created.
```

Comment:
Not a native speaker myself, but I think "that are need**ed** to be created" would be the correct spelling here.

Reply:
Fixed